meta/main.yml

```diff
@@ -1,16 +1,33 @@
-galaxy_info:
-  author: Emma Foley
-  description: TripleO collectd
-  company: Red Hat
+---
+# Copyright 2020 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
 
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  issue_tracker_url: http://github.com/tripleo-collectd-ansible-role/issues
 
+galaxy_info:
+  author: OpenStack
+  description: TripleO OpenStack role -- tripleo_collectd
+  company: Red Hat
   license: Apache-2.0
-
   min_ansible_version: 2.9
+  platforms:
+    - name: CentOS
+      versions:
+        - 7
```

Comment:
Do we actually support 7?

Reply:
That's anyone's guess, however, this is probably a copy-paste error